scripts/bootstrap-controllers.sh

```diff
@@ -0,0 +1,165 @@
+#!/usr/bin/bash
```

Comment:
Good point! I'll update the scripts to use the more portable shebang `#!/usr/bin/env bash`.